<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="Form" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="centralWidget" native="true">
     <widget class="QPushButton" name="button7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRes">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonDelen">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button9">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonPlus">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRaz">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>250</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonUmnog">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button0">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rezults">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>CE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="puttonPoint">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>,</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSqrt">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonKvadrat">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>x^2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonGip">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>1/x</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Form</class>
   <extends>QWidget</extends>
   <header>form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
